.github/workflows/release.yml

```diff
@@ -0,0 +1,83 @@
+name: release
+
+on: push
+
+jobs:
+  ubuntu-release:
+    runs-on: ubuntu-latest
+    env:
+      CARGO_HOME: /home/runner/.cargo
+      PREFIX: /home/runner/install
+    steps:
+      - name: Linux Cargo Cache
+        id: linux-cache-cargo
+        uses: actions/cache@v4
+        with:
+          path: /home/runner/.cargo
+          key: linux-cache-cargo
+      - name: Install botan
+        run: sudo apt install -y libbotan-2-dev
+      - uses: actions/checkout@v4
+      - run: sudo apt install cargo
```

Comment:
have a look at this action [here](https://github.com/plancksecurity/planck-core/blob/45fefece7d253342c88c89ec0b301dc6db8a9d07/.github/workflows/check-code-correctness.yml#L21)